airbyte-cdk/python/airbyte_cdk/sources/declarative/stream_slicers/datetime_stream_slicer.py

```diff
@@ -173,15 +174,13 @@ def _get_date(self, cursor_value, default_date: datetime.datetime, comparator) -
         cursor_date = self.parse_date(cursor_value or default_date)
         return comparator(cursor_date, default_date)
 
-    def parse_date(self, date: Any) -> datetime:
-        if date and isinstance(date, str):
-            if self.is_int(date):
-                return datetime.datetime.fromtimestamp(int(date)).replace(tzinfo=self._timezone)
-            else:
-                return dateutil.parser.parse(date).replace(tzinfo=self._timezone)
-        elif isinstance(date, int):
+    def parse_date(self, date: Union[str, datetime.datetime]) -> datetime.datetime:
```

Comment:
can't place the comment in the right spot so adding it here. Can you clarify in the doc string for `datetime_format` what the reference for formats is? eg add a link to it

Reply:
added a link in the docstring and in MinMaxDatetime's https://github.com/airbytehq/airbyte/pull/15326/files#diff-3a6ff2f6eca71b4e166b1c8c852becb143ad4e079f5308967ff52828e747f035R34